cmd/helm/testdata/testcharts/chart-with-subchart-notes/charts/subchart-with-notes/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Can you set this to v2? New chart format for Helm 3.

Reply:
Good call.  Just pushed.